myapp/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# app/controllers/sessions_controller.rb
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email])
```

Comment:
Can it use a index?

Reply:
I used it

Reply:
Sorry, if I look at schema file, I don't think there is an index at email column.
https://github.com/Fablic/training/blob/step16_siqi_fan/myapp/db/schema.rb

Reply:
sorry I added it now